src/dxvk/dxvk_device.cpp

```diff
@@ -26,6 +26,13 @@ namespace dxvk {
   
   
   DxvkDevice::~DxvkDevice() {
+    // If we are being destroyed during/after DLL process detachment
+    // from TerminateProcess, etc, our CS threads are already destroyed
+    // and we cannot synchronize against them.
+    // The best we can do is just wait for the Vulkan device to be idle.
```

Comment:
This comment seems misleading since we're not actually doing that? Would requrie a CS thread sync anyway.